receiver/sshcheckreceiver/metadata.yaml

```diff
@@ -0,0 +1,42 @@
+name: sshcheckreceiver
+
+resource_attributes:
+
+attributes:
+  ssh.endpoint:
+    description: Full SSH endpoint.
+    type: string
+
+metrics:
+  sshcheck.success:
+    description: The SSH server connected. 
+    enabled: true
+    gauge:
+      value_type: int
+    unit: 1
+    attributes: [ssh.endpoint]
+  sshcheck.duration:
+    description: Measures the duration of SSH check.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: false
+      value_type: int
+    unit: ms
+    attributes: [ssh.endpoint]
+  sshcheck.sftp_success:
```

Comment:
Based on suggestions and [discussion in httpcheck](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/14191#discussion_r987393250) of @jmacd and @djaglowski I plan to change how we model this to a non-monotonic cumulative sum.